### DISCUSSION TITLE
Running test on macOS

### DISCUSSION THREAD

**Discussion by bharsaklemukesh975** [ccache/ccache#1277] [Development]
Hi All,

I got the following error while running the test. any idea about the error or can someone please share the process run tests?

mukeshbharsakle@Mukeshs-MacBook-Pro ccache % test/run 
Compiler:         clang (/usr/bin/clang)
Compiler version: Apple clang version 14.0.3 (clang-1403.0.22.14.1)
CUDA compiler:    not available

Running test suite base.test/suites/base.bash: line 7: /Users/mukeshbharsakle/git/ccache/ccache: No such file or directory
test/run: line 172: /Users/mukeshbharsakle/git/ccache/ccache: No such file or directory

FAILED

Test suite:     base (line 8)
Test case:      Base case
Failure reason: Expected preprocessed_cache_hit to be 0, actual 

Actual statistics counters
==========================
test/run: line 51: /Users/mukeshbharsakle/git/ccache/ccache: No such file or directory

Test data and log file have been left in testdir/19491 / testdir/failed

**Comment by jrosdahl** [comment]
You need to build before running the tests. Example:

```
mkdir build
cd build
cmake ..
make
../test/run # or "make test"
```
